test/UnitSession.cpp

```diff
@@ -123,8 +123,8 @@ UnitBase::TestResult UnitSession::testHandshake()
         wsSession->poll(
             [&](const std::vector<char>& message)
             {
-                const std::string msg = Util::toString(message);
-                if (!Util::startsWith(msg, "error:"))
+                const std::string msg(std::string(message.begin(), message.end()));
```

Comment:
Would it work to just go with `const std::string msg(message.begin(), message.end())`?